Source/ChocolateyGui/Properties/Resources.nb.resx

```diff
@@ -0,0 +1,553 @@
+    <value>Detaljer</value>
+  </data>
+  <data name="PackageView_ButtonPin" xml:space="preserve">
+    <value>Fest</value>
+  </data>
+  <data name="Bootstrapper_UnhandledException" xml:space="preserve">
+    <value>Ubehandled Unntak</value>
+  </data>
+  <data name="ChocolateyCustomSchemeProvider_InvalidHtml" xml:space="preserve">
+    <value>&lt;h3&gt;Ugyldig HTML Streng&lt;/h3&gt;</value>
+  </data>
+  <data name="ChocolateyCustomSchemeProvider_UnknownHost" xml:space="preserve">
+    <value>&lt;h1&gt;Ukjent Vert&lt;/h1&gt;</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_Escalating" xml:space="preserve">
+    <value>Eskalering</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_ExceptionFormat" xml:space="preserve">
+    <value>
+Unntak: {0}</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_Exiting" xml:space="preserve">
+    <value>Avslutter</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_InstallFailedMessage" xml:space="preserve">
+    <value>Installeringen av pakken "{0}", versjon "{1}" feilet.
+Feil: {2}{3}</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_InstallFailedTitle" xml:space="preserve">
+    <value>Installering av pakke feilet.</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_PinFailedMessage" xml:space="preserve">
+    <value>Festingen av pakken "{0}", versjon "{1}" feilet.
+Feil: {2}{3}</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_PinFailedTitle" xml:space="preserve">
+    <value>Festing av pakke feilet.</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_UninstallFailedMessage" xml:space="preserve">
+    <value>Avinstalleringen av pakken "{0}", versjon "{1}" feilet.
+Feil: {2}{3}</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_UninstallFailedTitle" xml:space="preserve">
+    <value>Avinstallering av pakke feilet.</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_UnpinFailedMessage" xml:space="preserve">
+    <value>Løsningen av pakken "{0}", versjon "{1}" feilet.
+Feil: {2}{3}</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_UnpinFailedTitle" xml:space="preserve">
+    <value>Løsning av pakke feilet.</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_UpdateFailedMessage" xml:space="preserve">
+    <value>Oppdateringen av pakken "{0}", versjon "{1}" feilet.
+Feil: {2}{3}</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_UpdateFailedTitle" xml:space="preserve">
+    <value>Oppdatering av pakke feilet.</value>
+  </data>
+  <data name="LocalSourceViewModel_ConfigFiles" xml:space="preserve">
+    <value>Konfigurasjonsfiler (.config|*.config)</value>
+  </data>
+  <data name="LocalSourceViewModel_Packages" xml:space="preserve">
+    <value>Pakker</value>
+  </data>
+  <data name="LocalSourceViewModel_UpdateAvailableForChocolatey" xml:space="preserve">
+    <value>Det er en oppdatering tilgjengelig for chocolatey.</value>

```

Comment:
Det er brukt liten 'c' i den engelske varianten her, derfor bruk av liten 'c' i the norske burde nå vel være rett mener jeg.